documentation/internal/Level-1+-Reviews-and-Changes-Requested.md

```diff
@@ -0,0 +1,129 @@
+# Level 1+ Reviews and Changes Requested
+
+The [Level 1 Reviews](Level-1-Review-And-LOQ.md) explains how reviews with only 1 level works. Which is a subset of how a review with more levels (per stage) should work.
+
+The aim of this page is to outline the structure changes and flows:
+
+## Consolidation in agreement
+
+- Reviewer (Level 1) - reviewing an application
+- Reviewer (Level 1) - submit ~~`LOQ (List of Questions)`~~ `Conform` or `Non Conform` advise to upper level reviewer
+- Reviewer (Level 2) - reviewing the lower level review and `agree` with all advised decisions
+- Reviewer (Level 2) - submit a review decision - if user has `is_last_level` will send back to Applicant when `LOQ (List of Questions)` or change outcome and stage when submitting `Non Conform` or `Conform`. If consolidation not `is_last_level` submission of review goes to upper level reviewer (e.g. Level 3) and the same process repeats for Level 2 and Level 3 reveiwers.
+
+## Consolidation in disagreement (Changes-requested)
+
+- Reviewer (Level 1) - reviewing an application
+- Reviewer (Level 1) - submit ~~`LOQ (List of Questions)`~~ `Conform` or `Non Conform` advise to upper level reviewer
+- Reviewer (Level 2) - consolidate lower level review(s) and `disagree` with some advised decisions
+- Reviewer (Level 2) - submit `Changes Requested` back to reviewer (e.g. Level 1)
+- Reviewer (Level 1) - makes changes in response to `Changes Requested` and re-submit to upper level reviewer.
+
+### Schema changes
+
+![Level-1+-Review-Structure-Changes](images/Level-1+-Review-Structure-Changes.png)
+
+#### Consolidation
+
+- There is just one specific progress for a consolidation of lower level reviews (`consolidationProgress`).
```

Comment:
Not sure exactly what you mean with sentence? Did you mean "process" rather than "progress"?

Reply:
Slightly changed sentence. Please, see if it's better in new [PR](#909)